includes/class-migration.php

```diff
@@ -798,6 +800,69 @@ private static function add_notification_defaults() {
 		\add_option( 'activitypub_mailer_new_dm', '1' );
 	}
 
+	/**
+	 * Add a default extra field for the user.
+	 */
+	private static function add_default_extra_field() {
+		$users = \get_users(
+			array(
+				'capability__in' => array( 'activitypub' ),
+			)
+		);
+
+		$title   = __( 'Powered by', 'activitypub' );
+		$content = __( 'WordPress ❤️', 'activitypub' );
+
+		// Add a default extra field for each user.
+		foreach ( $users as $user ) {
+			// check if the extra field already exists.
+			$extra_field = get_posts(
+				array(
+					'post_type'      => Extra_Fields::USER_POST_TYPE,
+					'author'         => $user->ID,
+					'posts_per_page' => 1,
+				)
+			);
+
+			if ( $extra_field ) {
+				continue;
+			}
+
+			wp_insert_post(
```

Comment:
Do you anticipate this running any other time, aside from first activation? If not, I wouldn't bother looking up existing posts and instead just add them.

Reply:
I want to get sure to only add the default if no other extra-fields are added already. This also helps with tests 🫣